dashboard/app/models/concerns/user/retainable.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
```

Comment:
I think this line is automatically added when creating a new Ruby file in some IDEs. I typically remove it, but I'm curious if we should keep it.